File: BAEKJOON/Bronze4/1267/main.py
def Call30(time):
    return ((time // 30) + 1) * 10


def Call60(time):
    return ((time // 60) + 1) * 15


call = int(input())

arr = list(map(int, input().split()))

Y_sum = 0
M_sum = 0

for i in arr:
    Y_sum += Call30(i)
    M_sum += Call60(i)

if Y_sum < M_sum:
    print('Y', Y_sum)
elif Y_sum > M_sum:
    print('M', M_sum)
else:
    print('Y', 'M', Y_sum)

File: BAEKJOON/Bronze4/1267/main.c
#include <stdio.h>

int main()
{
    int N;
    int arr[20];
    scanf("%d", &N);
    for (int i = 0; i < N; i++)
    {
        scanf("%d", &arr[i]);
    }

    // Y의 요금제 구하기
    int Y = 0;
    for (int i = 0; i < N; i++)
    {
        Y += (arr[i] / 30 + 1) * 10;
    }
    // M의 요금제 구하기
    int M = 0;
    for (int i = 0; i < N; i++)
    {
        M += (arr[i] / 60 + 1) * 15;
    }

    printf("%d %d", Y, M);

    return 0;
}